File: age_verifier.js
var ageVerifier = function(ages) {
    ages.forEach(function (age) {
        if (age < 21) {
            console.log("too young");
        } else {
            console.log("proceed");
        }
    });
}

var arr_of_ages = [10, 20, 40, 15, 30];
ageVerifier(arr_of_ages);

File: age_verifier.rb
def age_verifier(ages)
  ages.each do |age|
    if age < 21
      puts "too young"
    else
      puts "proceed"
    end
  end
end

arr_of_ages = [10,15,25,30,35]
age_verifier(arr_of_ages)
